template.yaml

```diff
@@ -12,8 +12,25 @@ Parameters:
     Type: String
     Default: 'dlr-nva-thumbnails'
     Description: Name of output bucket. Thumbnails created by this function will be stored here. This bucket is also setup to be public read.
+  FfmpegLambdaLayerApplicationID:
```

Comment:
If there is only one allowed value, why do we have to have it as a parameter in master-pipelines? Why can't we use it directly here? Are we going to be able to select between different SAR applications depending on env at some point?

Reply:
Thought I needed to put string dependencies in the master pipelines. Can change it to only be used only in this template.